doc/examples/cython/.rayproject/project.yaml

```diff
@@ -0,0 +1,46 @@
+# This file is generated by `ray project create`.
+
+name: ray-example-cython
+
+description: "Example of how to use Cython with ray"
+tags: ["ray-example", "cython"]
+documentation: https://ray.readthedocs.io/en/latest/advanced.html#cython-code-in-ray
+
+cluster: .rayproject/cluster.yaml
+
+environment:
+  requirements: .rayproject/requirements.txt
+
+  shell: # Shell commands to be ran for environment setup.
+    - pip install -e .
+
+commands:
+  - name: example1
+    command: python cython_main.py example1
+    help: "Run a simple Cython function"
+  - name: example2
+    command: python cython_main.py example2
+    help: "Run a simple recursive Cython function"
+  - name: example3
+    command: python cython_main.py example3
+    help: "Run a simple typed Cython function"
+  - name: example4
+    command: python cython_main.py example4
+    help: "Run a Cython defined with cpdef"
+  - name: example5
+    command: python cython_main.py example5
+    help: "Run a Cython defined with cdef"
+  - name: example6
+    command: python cython_main.py example6
+    help: "Run a simple Cython class"
+  - name: example7
+    command: python cython_main.py example7
+    help: "Run Cython with function from BrainIAK"
+  - name: example8
+    command: python cython_main.py example8
+    help: "Run Cython with BLAS"
+
+output_files: [
```

Comment:
Should this be here?

Reply:
You mean in general or only for the cython example? It also uses ray, so it makes sense. It will allow us to save the logs in the future :)